graphql-api/src/main/resources/stream-registry.graphql

```diff
@@ -1,49 +1,37 @@
-type KeyValue {
-  key: String!
-  value: String!
-}
+scalar Tags
+scalar Config
 
 type Domain {
   name: String!
```

Comment:
do these objects need an id as well as a name, or are they one and the same?

Reply:
GraphQL has an `ID` type but each entity has it's own composite primary key. domain is the name, schema is the schema name and domain name, and stream is the stream name, domain name and stream version.